style/index.css

```diff
@@ -50,3 +50,177 @@
   padding: 12px 0 4px 12px;
   text-transform: uppercase;
 }
+
+.jp-TableOfContents-codeContainer {
+  overflow: hidden;
+}
+
+.jp-TableOfContents-code {
+  font-size: 9px;
+  max-height: 70px;
+}
+
+.cm-toc .CodeMirror {
+  font-size: 9px;
+  z-index: 0;
+  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
+  border-radius: 0px;
+  background: var(--jp-cell-editor-background);
+  max-width: 100%;
+  max-height: 81px;
+}
+
+.toc-code-span {
+  width: 100%;
+  max-width: 100%;
+  overflow: hidden;
+}
+
+.cm-toc .CodeMirror-scroll {
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.markdown-cell {
+  overflow-x: hidden;
+  overflow-y: hidden;
+  max-height: 22px;
+}
+
+.markdown-cell code {
+  font-size: 9px;
+}
+
+.CodeMirror-scroll::-webkit-scrollbar-track {
+  background-color: transparent;
+}
+
+.dropdown-menu-list {
+  z-index: 3;
+  position: fixed;
+  list-style-type: none;
+  margin: 0px;
+  padding: 0px;
+  background-color: white;
```

Comment:
This class should be updated to use the JupyterLab theme variables. Otherwise it is hard to read in the dark theme (and similar).